test/yagna/helpers/activity.py

```diff
@@ -0,0 +1,73 @@
+"""Activity helpers."""
+
+import os
+from pathlib import Path
+
+from goth.runner import Runner
+
+
+def vm_exe_script(runner: Runner, output_file: str):
```

Comment:
Let's add a default value to `output_file` like in the case of `wasi_exe_script` (e.g. `output.png`).